services/core-api/app/api/projects/project_summary/models/project_summary_authorization.py

```diff
@@ -18,26 +22,191 @@ class ProjectSummaryAuthorization(SoftDeleteMixin, AuditMixin, Base):
+            },
+            'project_summary_authorization_type': {
+                'required': True,
+                'type': 'string'
+            }
+        }
+        # all AMS authorizations
+        common_ams_schema = common_schema | {
+            'project_summary_permit_type': {
+                'required': True,
+                'type': 'list',
+                'items': [{
+                    'type': 'string',
+                    'allowed': ['AMENDMENT', 'NEW']
+                }]
+            },
+            'authorization_description': {
+                'required': True,
+                'type': 'string',
+                'maxlength': 4000
+            },
+            'exemption_requested': {
+                'required': True,
+                'type': 'boolean'
+            },
+            'project_summary_authorization_type': {
+                'required': True,
+                'type': 'string',
+                'allowed': ams_auth_types_list
+            }
+        }
+
+        ams_amendment_schema = common_ams_schema | {
+            'existing_permits_authorizations': {
+                'required': True,
+                'type': 'list',
+                'schema': {'type': 'string'},
+                'items': [{
+                    'required': True,
+                    'type': 'string',
+                    'regex': '^[\d]{4}$'
+                }]
+            },
+            'amendment_severity': {
+                'required': True,
+                'type': 'string',
+                'allowed': ['SIG', 'MIN']
+            },
+            'amendment_changes': {
+                'required': True,
+                'type': 'list',
+                'allowed': ['ILT', 'IGT', 'DDL', 'NAM', 'TRA', 'MMR', 'RCH', 'OTH']
+            },
+            'is_contaminated': {
+                'required': True,
+                'type': 'boolean'
+            }
+        }
+        ams_new_schema = common_ams_schema | {
+            'new_type': {
+                'required': True,
+                'type': 'string',
+                'allowed': ['PER', 'APP']
+            }
+        }
+
+        v = Validator(common_schema, purge_unknown=True)
+        if is_ams:    
+            permit_type = authorization.get('project_summary_permit_type')
+
+            if not permit_type or not isinstance(permit_type, list) or not permit_type[0]:
+                return f'Invalid authorization type {permit_type}'
+            if permit_type[0] == 'AMENDMENT':
+                v = Validator(ams_amendment_schema, purge_unknown=True)
+            elif permit_type[0] == 'NEW':
+                v = Validator(ams_new_schema, purge_unknown=True)
+            else:
+                v = Validator(common_ams_schema, purge_unknown=True)        
+        elif authorization.get('project_summary_authorization_type') == 'OTHER':
+            v = Validator(other_schema, purge_unknown=True)
+
+        if not v.validate(authorization):
```

Comment:
The errors aren't terribly _pretty_ (by default...) but they get the job done. I found it easier to use this than to do all the `if`'s and `len`'s and `not in`'s I'd be doing otherwise. It's not specific about which authorization is causing the issue, for the reason that I didn't want to cause a 500 trying to get an attribute that may be causing the 400, so just logged the whole record on the BE if it's invalid